Add SJ release notes for 2.4 (#479)
NCI-GDC/gdc-docs: docs/Data_Portal/Release_Notes/Data_Portal_Release_Notes.md

* __Release Date__:  March 13, 2025

### New Features and Changes

> * __New Analysis Tool:  Single-Cell RNAseq__:

Reviewer: @siosonel Add a return above this line and I'll be ready to merge this.  I'll approve for now.

Author: Done. Thanks for the review!

* Updated the Repository and Cart to display Sample Sheet and Metadata downloads as separate buttons for better visibility. <!--PEAR-2232-->
* Minor text and styling improvements. <!--PEAR-452/2298/2341/2323/2253/2290-->

> * __Single-Cell RNAseq__:

Reviewer: ```suggestion
* __New Analysis Tool:  Single-Cell RNAseq__:
```

I suggest putting this at the very top as analysis tools are a big part of the Data Portal.  Also add a return under "### New Features and Changes" (just to save us a commit).

    * Visualize single-cell RNA-Seq data with cluster plots and gene expression overlays.
* __OncoMatrix__:

>     * By default hide “splice region” consequence mutations.

Reviewer: ```suggestion
    * By default hide "splice region" consequence mutations.
```
I'll try to look for these, but remove any curly quotes that I don't catch.

Author: Thanks, fixed

>     * Support 5-category CNV; in OncoPrint mode, plot a white border around SSM to better distinguish it from CNV when columns are wide enough

Reviewer: ```suggestion
    * Support 5-category CNV: in OncoPrint mode, plot a white border around SSM to better distinguish it from CNV when columns are wide enough
```
A colon seems more appropriate here, but let me know if that wasn't your intention. It's probably cool either way.

    * Continuous variable row label menu supports option to edit bar height and color.
    * Allow user to modify colors from the legend.

>     * Show white border for SNV-Indel cell in OncoPrint mode when CNV data is also rendered.

Reviewer 2: duplicate

* __Gene Expression Clustering__:
    * Avoids showing genes with no expression values (due to an issue of ProteinPaint GENCODE v36 data that affects only certain genes; data issue will be fixed at the next release)
    * Make zscore transformation a checkbox option in Clustering menu.
* __Cohort MAF__:

>     * Allowed to sort the table of MAF files. Resolved a bug to allow the correct set of files to be returned after sorting the table (the bug was not present in the production environment). Used barplot to display MAF file size and disabled sorting for now.

Reviewer: You can probably remove the bug comment if it wasn't in prod.

* Data in arrays are now included in the clinical and biospecimen TSVs. <!--SV-2564/PEAR-2339-->
* Fixed an intermittent issue where __Quick Search__ navigation would randomly fail, preventing users from reaching the expected page. <!--PEAR-2294-->

* __Gene Expression Clustering__:

>     * Show value when it equals to 0 in hover over tooltip and click menu.

Reviewer: ```suggestion
    * Show value when it's equals to 0 in the hover over tooltip and click menu.
```

* __OncoMatrix__:

>     * Show value when it equals to 0 in hover over tooltip and click menu.

Reviewer: ```suggestion
    * Show value when it's equals to 0 in the hover over tooltip and click menu.
```

    * When the survival plot is zoomed in and an image is downloaded, the curves within the image may extend beyond the y-axis. <!--SV-2348-->
* __Gene Expression Clustering__:
    * The tool allows deleting the gene expression group and displays an uninformative error message after submitting the deletion.
* __Cohort MAF__:
    * A downloaded file may be corrupted if the server data processing is terminated after 5 minutes in order to conserve server resources.

>     There will be a red banner above the MAF contorls to indicate the termination.

Reviewer: ```suggestion
    * There will be a red banner above the MAF controls to indicate the termination.
```

Author: The lack of asterisk before "There will be .." is intentional. There is no extra space in the preceding line/list item, so this new line will be considered part of the previous list entry/bullet item.
